ci/vars-zuul-common.yml

```diff
@@ -9,3 +9,5 @@ sgo_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/smart-g
 sg_core_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/sg-core'].src_dir }}"
 sg_bridge_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/sg-bridge'].src_dir }}"
 prometheus_webhook_snmp_dir: "{{ ansible_env.HOME }}/{{ zuul.projects['github.com/infrawatch/prometheus-webhook-snmp'].src_dir }}"
+__service_telemetry_storage_persistent_storage_class: "crc-csi-hostpath-provisioner"
+__service_telemetry_storage_ephemeral_enabled: false
```

Comment:
The other files simply dropped this value (I assume to use the default). Would that be applicable here as well?

Reply:
Can definitely be dropped, it defaults to false here: https://github.com/infrawatch/service-telemetry-operator/blob/master/build/stf-run-ci/defaults/main.yml#L18